apps/frontend/src/app/backups-overview/backups/backups/backups.component.html

```diff
@@ -35,6 +35,10 @@ <h1 class="header">Backups Overview</h1>
   </div>
 </div>
 
+<div>
```

Comment:
I would add a `clr-row `class to define the size of this component in the same way as the other components in the dashboard.